elastic/kibana
# [ResponseOps][Alerting] Ability to bulk update API keys for alerting rules

Resolves https://github.com/elastic/kibana/issues/138011
## Summary

Allows users to bulk update API keys for alerting rules in the rules table.


### Checklist



### To verify

- Select multiple rules and select the `Manage rules` button
- Select Update API Keys and verify that the keys are updated properly
